bricks/template/__brick__/{{project_name.snakeCase()}}/lib/main_view_model.dart

```diff
@@ -0,0 +1,25 @@
+import 'dart:async';
+import 'package:{{project_name.snakeCase()}}/main_view_state.dart';
+import 'package:{{project_name.snakeCase()}}/usecases/base/base_use_case.dart';
+import 'package:{{project_name.snakeCase()}}/usecases/user/get_users_use_case.dart';
+import 'package:{{project_name.snakeCase()}}/model/user.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+class MainViewModel extends StateNotifier<MainViewState> {
+  final GetUsersUseCase _getUsersUseCase;
+
+  final StreamController<List<User>?> _usersStream = StreamController();
```

Comment:
Can we get rid of nullable?

```suggestion
  final StreamController<List<User>> _usersStream = StreamController();
```